### ISSUE TITLE
i18n support

### ISSUE THREAD

**Issue by batwicket** [matthiasn/systems-toolbox#17]
Allow developers to easily localize app.

Something like this might be a viable approach: https://github.com/ptaoussanis/tower

**Comment by matthiasn** [comment]
Good idea, this will be very useful in the project I'm working on at the moment. Thanks
